# Metagenomics_Pathogen_Discovery

Abstract

Background: Respiratory tract infections (RTIs) are a major cause of illness and death in children, with tuberculosis (TB) playing a significant role. This study uses metagenomic next- generation sequencing (mNGS) to explore pathogen detection in pediatric patients.
Results: Nasopharyngeal aspirate samples obtained from the Umoya cohort included four pediatric patients: two with confirmed TB, one with unconfirmed TB, and a healthy control. Each patient provided two samples, one treated with Benzonase and one untreated. Benzonase- treated samples yielded an average of 35.3 million reads, while untreated samples had 24.8 million. Despite increased microbial reads in treated samples, host DNA still comprised 90% of the total, indicating limited depletion. As for pathogen discovery, Patient 1 showed the possibility of a bacterial-viral co-infection, Patient 2â€™s sequencing failed, and Patient 3 revealed additional bacterial species possibly contributing to TB. Patient 4 had a healthy microbiome, though Enterovirus D was detected in one sample. Pathogens identified included Haemophilus influenzae, Streptococcus pneumoniae, Cytomegalovirus, and Parvovirus B19, with distinct differences between symptomatic patients and the control.
Conclusions: This study underscores the complexity of microbial interactions in pediatric RTIs and highlights challenges in host DNA depletion. Optimizing protocols is crucial for improving microbial detection and diagnostic accuracy, particularly in children with TB, to inform better therapeutic strategies and outcomes.
